pyvista/core/dataobject.py

```diff
@@ -676,7 +676,7 @@ def copy_structure(self, dataset: _vtk.vtkDataSet) -> None:
         >>> target.plot(show_edges=True)
 
         """
-        self.CopyStructure(dataset)
+        self.CopyStructure(dataset) if dataset is not self else None
```

Comment:
```suggestion
        self.CopyStructure(dataset) if dataset is not self
```

Reply:
I would also prefer this but it's not valid syntax, `else` is always required unfortunately

Reply:
Too bad Python is a bit redundant.